src/prpy/perception/chisel.py

```diff
@@ -0,0 +1,139 @@
+import logging
+from base import PerceptionModule, PerceptionMethod
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+class ChiselModule(PerceptionModule):
+    
+    def __init__(self, env, mesh_client=None, service_namespace=None,
+               detection_frame=None, destination_frame=None, reference_link=None):
```

Comment:
What is `detection_frame`? It's missing a docstring.

If this is the camera frame, please replace this with the `frame_id` from the `CameraInfo` message.